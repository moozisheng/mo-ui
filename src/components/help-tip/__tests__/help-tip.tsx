import '@testing-library/jest-dom';
import { cleanup, render } from '@testing-library/react';
import React from 'react';
import HelpTip from '../index';

afterEach(cleanup);

test('renders TagList with base use', () => {
  const { container } = render(
    <HelpTip help="这里是帮助提示信息">帮助提示</HelpTip>,
  );
  expect(container).toMatchSnapshot();
});

test('renders TagList with base use', () => {
  const { container } = render(
    <HelpTip help="这里是帮助提示信息" iconOnly>
      帮助提示
    </HelpTip>,
  );
  expect(container).toMatchSnapshot();
});
