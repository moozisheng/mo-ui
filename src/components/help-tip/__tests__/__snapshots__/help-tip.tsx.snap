// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TagList with base use 1`] = `
<div>
  <span
    class="help-tip inline-flex items-center cursor-help"
  >
    帮助提示
    <span
      aria-label="info-circle"
      class="anticon anticon-info-circle text-inherit opacity-75 ml-0.8 mr-0.2 flex cursor-help"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="info-circle"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
        />
        <path
          d="M464 336a48 48 0 1096 0 48 48 0 10-96 0zm72 112h-48c-4.4 0-8 3.6-8 8v272c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V456c0-4.4-3.6-8-8-8z"
        />
      </svg>
    </span>
  </span>
</div>
`;

exports[`renders TagList with base use 2`] = `
<div>
  <span
    class="help-tip inline-flex items-center"
  >
    帮助提示
    <span
      aria-label="info-circle"
      class="anticon anticon-info-circle text-inherit opacity-75 ml-0.8 mr-0.2 flex cursor-help"
      role="img"
    >
      <svg
        aria-hidden="true"
        data-icon="info-circle"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
        />
        <path
          d="M464 336a48 48 0 1096 0 48 48 0 10-96 0zm72 112h-48c-4.4 0-8 3.6-8 8v272c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V456c0-4.4-3.6-8-8-8z"
        />
      </svg>
    </span>
  </span>
</div>
`;
